tests/python/gaia-ui-tests/gaiatest/tests/accessibility/system/test_a11y_cards_view.py

```diff
@@ -96,7 +96,7 @@ def test_a11y_cards_view(self):
         # Open an app by clicking its icon
         cards[0].a11y_click_app_icon()
         cards_view.wait_for_cards_view_not_displayed()
-        self.assertEqual(self.apps.displayed_app.name, self._test_apps[0])
+        self.assertEqual(self.apps.displayed_app.manifest_url, 'app://calendar.gaiamobile.org/manifest.webapp')
```

Comment:
Use Calendar(self.marionette).manifest_url instead of hard-coding these urls.
You need to also do the calender import then, of course.